pom.xml

```diff
@@ -130,7 +130,13 @@
         <dependency>
             <groupId>org.sourcelab</groupId>
             <artifactId>kafka-connect-client</artifactId>
-            <version>3.1.0</version>
```

Comment:
Updated the kafka-connect-version, as it brings jackson-databind as a dependency. And jackson-databind needed to be updated to fix the security issues.

Reply:
what was the jackson-databind vulnerability? Just the max hierarchy depth default taht could cause denial of service attacks?

Reply:
These were the two vulnerabilities
- https://avd.aquasec.com/nvd/2022/cve-2022-42003/
- https://avd.aquasec.com/nvd/2022/cve-2022-42004/

Reply:
Yes - thanks